### DISCUSSION TITLE
Conteúdos da DB 192

### DISCUSSION THREAD

**Discussion by arturcorreaf** [YuriAlessandro/gerador-ficha-tormenta20#352] [Geral]
**Poderes de Caçador**

- Armadilha Alquímica
- Armadilha Oportunista
- Avanço do Predador
- Carnificina
- Corte Arterial
- Defesa Hostil
- Disparo Distrator
- Disparo Retentor
- Elo com a Natureza Maior
- Encurralar Presa
- Explorador Marcial
- Flecheiro
- Improvisar Munição
- Lâminas Guardiãs
- Lanceiro
- Matador de Gigantes
- Olhar Vigilante
- Predador Solidário
- Mãos de Curandeiro
- Sede de Sangue
- Sempre Alerta
- Tiro em Linha
- Último Sangue

**Novas melhorias para munições**

- Agulha
- Assobiadora
- Cauda-de-Andorinha
- Crescente de Lena
- Fósforo
- Incendiária
- Punhal
- Rombuda

**Comment by YuriAlessandro** [comment]
Boa, obrigado por já deixar assim organizadinho. Vou mandar para lá ainda hoje.

**Comment by YuriAlessandro** [comment]
Adicionado, está no ar. Muito obrigado.
